src/js/pages/jobs/JobConfiguration.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import ChronosStore from '../../stores/ChronosStore';
+import DescriptionList from '../../components/DescriptionList';
+
+class JobConfiguration extends React.Component {
+  render() {
+    let job = ChronosStore.getJob(this.props.jobID);
```

Comment:
Can we pass in the job as a property, instead of fetching it from the store again?